Source/KakapoDB.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  KakapoDB.swift
+//  KakapoExample
+//
+//  Created by Joan Romano on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+protocol KStorable {
+    var id: Int { get }
```

Comment:
Why not an Int? Also, since we are clearly _not_ building a database, do we really need to care about thread safety regarding that id? Are we accessing this from multiple threads in our unit tests environments?